app/models/post.rb

```diff
@@ -15,4 +15,5 @@ class Post < ApplicationRecord
   scope :view_count_sort, ->{order view_count: :desc}
   scope :select_field, ->{select :id, :title, :content, :user_id, :view_count, :created_at}
   scope :of, ->(user){where user: user}
+  scope :search, ->(keyword){where "title LIKE ? OR content LIKE ?", "%#{keyword}%", "%#{keyword}%"}
```

Comment:
em thử theo cách này thì rubocop báo 2 lỗi là dùng lambda method thay cho -> và thêm () vào keyword ạ.
Em có làm theo hướng dẫn trên trang https://github.com/bbatsov/ruby-style-guide#lambda-multi-line (Rubocop gợi ý) và sửa thành 
`scope :search, lambda do |keyword|`
`  where.....`
`end`
thì lúc chạy app lại báo `tried to create Proc object without a block`
=> dùng `lambda` thì chạy app lỗi nhưng rubocop pass
     dùng `->` thì chạy app bình thường nhưng rubocop failed ạ
Giờ em phải sửa như thế nào ạ.